test-harness/specs/proxy/proxy.get-with-body.txt

```diff
@@ -0,0 +1,20 @@
+====test====
+Making a GET request with request body through the proxy server rejects with 501
+====spec====
+swagger: "2.0"
+paths:
+  /status/204:
+    get:
+      produces:
+        - text/plain
+      responses:
+        204:
+          description: No Content
+====server====
+proxy -p 4010 ${document} http://httpbin.org
+====command====
+curl -i http://localhost:4010/status/204 -X GET --header 'Content-Type: application/json' --data '{}'
```

Comment:
I've never tried to do this before.  Does `'{}'` get serialized to a zero-length request body?

Reply:
yea - its a verbatim string here - so the request body is just `{}`